scene/animation/animation_node_state_machine.cpp

```diff
@@ -963,6 +963,7 @@ bool AnimationNodeStateMachinePlayback::_transition_to_next_recursive(AnimationT
 
 		// Update current status.
 		_set_current(p_state_machine, next.node);
+		p_tree->emit_signal("animation_playback_node_changed", next.node);
```

Comment:
This is an inappropriate signal firing. At least it should be in `_set_current()`.